packages/@instana-integration/go/README.md

```diff
@@ -44,3 +44,23 @@ Below are the resource attributes that are currently supported by this integrati
 |----------------------------|-------|------------------------|
 | service.name               | string  | This attribute is used to describe the entity name.    |
 | service.instance.id        | string  | This attribute is used to describe the entity ID of the current object.  |
+
+### Installation and Usage
+
+With Instana CLI [stanctl-integration](https://github.com/instana/observability-as-code) for integration package management, you can manage the lifecycle of this package by downloading and importing into Instana.
+
+Downloading the package:
+
+```shell
+$ stanctl-integration download --package @instana-integration/go
+```
+
+Importing the dashboard into Instana:
+
+```shell
+$ stanctl-integration import --package @instana-integration/go \
+  --server $INSTANA_SERVER \
+  --token $API_TOKEN \
+  --set servicename=$SERVICE_NAME \
```

Comment:
Do we have these two params defined in the dashboard? Same applies to nodejs and opentelemetry-demo package.

Reply:
Yes, that's why we need the `Installation and Usage` section here.